howso/utilities/feature_attributes/base.py

```diff
@@ -617,14 +617,20 @@ def _process(self,  # noqa: C901
                         'format will be ignored.')
                 elif feature_type == FeatureType.TIME:
                     feature_attributes[feature_name] = (
-                        self._infer_datetime_attributes(feature_name, user_dt_format))
+                        self._infer_time_attributes(feature_name, user_dt_format))
                 elif isinstance(user_dt_format, str):
                     # User passed only the format string
-                    feature_attributes[feature_name] = {
-                        'type': 'continuous',
-                        'data_type': 'formatted_date_time',
-                        'date_time_format': user_dt_format,
-                    }
+                    # First see if it is likely a time-only feature
+                    if any(identifier in user_dt_format
+                           for identifier in ['%I', '%H', '%M', '%S', '%f', '%p']):
+                        feature_attributes[feature_name] = (
```

Comment:
Should check there are no date identifiers, otherwise this will potentially incorrectly assume its time only